### PR TITLE
ECDC-1653: fail build doxygen warning

### PR DESCRIPTION
**Pull request by kmurica**
### Issue

*Specify the associated JIRA ticket, e.g. DM-1234.*

### Description of work

*Add a description of the work done. The aim is provide information to help the reviewer review the PR efficiently.
See [CONTRIBUTING.MD](CONTRIBUTING.MD) for more information on code standards etc.*

### Nominate for Group Code Review

- [ ] Nominate for code review 

### Reminder

*Changes should be documented in `changes.md`*

### CODE REVIEW

**Comment by kmurica** [comment]
@SkyToGround Proposed solution. Feel free to have a look at my changes to JenkinsFile and doxygen config.
We (or as agreed: I ) will have to address the missing documentation in following ticket:
https://jira.esss.lu.se/browse/ECDC-1654
So I suggest that we proceed to use this branch until all warnings are fixed for doxygen.

File: Jenkinsfile [hunk 1]
```diff
@@ -271,10 +271,41 @@ builders = pipeline_builder.createBuilders { container ->
...
+        }
+
+        int numFailedCases = failingCases.size()
+        if(numFailedCases) {
+            doxygenStepFailed = true
+            println "Doxygen failed to generate HTML documentation due to following warnings:"
+            for(line in failingCases) {
+                println line
+            }
+            println "Total number of Doxygen warnings that needs to be corrected is $numFailedCases."
```
Comment (SkyToGround) [inline Jenkinsfile:300]: I think this line should be printed before the actual warning messages so as to make it easier for you to get an overview on how much work you have to do.

**Comment by SkyToGround** [comment]
@kmurica This looks good and yes, I agree.

**Comment by kmurica** [comment]
> @kmurica This looks good and yes, I agree.

Great. I will move ECDC-1653 to done then (changed the printout order according your comment) and continue on the documentation in the other ticket (but probably not right away :) ).

**Pull request merged**